### DISCUSSION TITLE
How to change or remove names displayed in the kills feed

### DISCUSSION THREAD

**Discussion by Petkoww** [fraxiinus/ReplayBook#227] [General]
Hello, my question is how can I remove or change the displayed names in the kills feed?

Example:    "Sion has killed Sylas" or " Player1 has killed Player2 " ?

Thanks in advance :)

**Comment by fraxiinus** [comment]
Anything that happens in game is outside the scope of ReplayBook, the program can't do anything there.
There is this setting in the client you can try.

![image](https://user-images.githubusercontent.com/8365233/219882261-6237f4ce-63cd-44cc-ad8f-c09e1033d69a.png)
